ci/build.py

```diff
@@ -138,6 +163,10 @@ def script_name() -> str:
                         help="List platforms",
                         action='store_true')
 
+    parser.add_argument("--docker_run",
```

Comment:
Could you rename it to "print_docker_run"? To me, the current name indicates we're actually trying to run docker